x-pack/plugins/maps/common/descriptor_types/source_descriptor_types.ts

```diff
@@ -42,6 +42,7 @@ export type AbstractESSourceDescriptor = AbstractSourceDescriptor & {
   geoField?: string;
   applyGlobalQuery: boolean;
   applyGlobalTime: boolean;
+  respondToForceRefresh: boolean;
```

Comment:
How about `applyForceRefresh` instead of `respondToForceRefresh`? The flag is used in the same way as `applyGlobalQuery` and `applyGlobalTime` so I think it makes sense that the naming convention follows as well.